content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -125,30 +148,51 @@ you'll need to use `replace` rather than `create` in the following command.
     ---
     {{< /text >}}
 
-    > The corresponding `subset` definitions can be displayed using `istioctl get destinationrules -o yaml`.
+1. Display the corresponding `subset` definitions:
 
-    Since config propagation is eventually consistent, wait a few seconds for the virtual services to take effect.
+    {{< text bash >}}
+    $ istioctl get destinationrules -o yaml
+    {{< /text >}}
+
+You have configured Istio to route to the `v1` version of the Bookinfo microservices,
+including the `ratings` service.
+
+## Test the new routing configuration
+
+You can easily test the new configuration by once again refreshing the `/productpage`
+of the Bookinfo app.
+
+1.  Open the Bookinfo site in your browser. The URL is `http://$GATEWAY_URL/productpage`, where `$GATEWAY_URL` is the External IP address of the ingress, as explained in
+the [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port) doc.
+
+    Notice that the reviews part of the page displays with no rating stars, no
+    matter how many times you refresh. This is because you configured Istio to route
+    all traffic for the reviews service to the version `reviews:v1` and this
+    version of the service does not access the star ratings service.
 
-1.  Open the Bookinfo URL (`http://$GATEWAY_URL/productpage`) in your browser. Recall that `GATEWAY_URL`
-    should have been set using [these instructions](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port)
-    when the Bookinfo sample was deployed.
+You have successfully accomplished the first part of this task: route traffic to one
+version of a service.
 
-    You should see the Bookinfo application productpage displayed.
-    Notice that the `productpage` is displayed with no rating stars since `reviews:v1` does not access the ratings service.
+## Route based on user identity
 
-1.  Route a specific user to `reviews:v2`
+Next, you will change the route config so that all traffic from a specific user
+is routed to a specific service version. In this case, all traffic from a user
+named Jason will be routed to the service `reviews:v2`.
 
-    Lets enable the ratings service for test user "jason" by routing productpage traffic to
-    `reviews:v2` instances.
+Remember, `reviews:v2` is the version that includes the star ratings feature.
+
+1. Run the following command to enable the user-based routing:
 
     {{< text bash >}}
     $ istioctl replace -f @samples/bookinfo/networking/virtual-service-reviews-test-v2.yaml@
     {{< /text >}}
 
-    Confirm the rule is created:
+1. Confirm the rule is created:
 
-    {{< text bash yaml >}}
```

Comment:
Why this change? We want to show the output of a command in the same block as the command line itself.